app/src/main/java/com/xplore/paymobile/ui/merchantselection/PostLoginMerchantSelectFragment.kt

```diff
@@ -0,0 +1,53 @@
+package com.xplore.paymobile.ui.merchantselection
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.activityViewModels
+import androidx.fragment.app.viewModels
+import androidx.lifecycle.lifecycleScope
+import androidx.navigation.fragment.findNavController
+import com.xplore.paymobile.R
+import com.xplore.paymobile.databinding.FragmentPostLoginMerchantSelectBinding
+import com.xplore.paymobile.ui.base.BaseFragment
+import dagger.hilt.android.AndroidEntryPoint
+import kotlinx.coroutines.launch
+
+@AndroidEntryPoint
+class PostLoginMerchantSelectFragment : BaseFragment() {
+
+    private var _binding: FragmentPostLoginMerchantSelectBinding? = null
+    private val binding get() = _binding!!
+
+    override val hasBottomNavigation: Boolean = false
+
+    private val viewModel by viewModels<PostLoginMerchantSelectViewModel>()
```

Comment:
i think this viewModel is useless here. Can't be deleted from here together with the PostLoginMerchantSelectViewModel class?